script.js

```diff
@@ -0,0 +1,74 @@
+// const addBookBtn = document.querySelector('.add');
```

Comment:
- [ ] it would be better to remove unused commented code in order to keep you file clean and pro